openquakeplatform/openquakeplatform/exposure/views.py

```diff
@@ -82,8 +83,8 @@
 """
 
 #: The maximum bounding box area which can be exported.
-MAX_EXPORT_AREA_SQ_DEG = 4  # 2 * 2 degrees, for example
-MAX_TOT_GRID_COUNT = 100000
+MAX_EXPORT_AREA_SQ_DEG = settings.EXPOSURE_MAX_AREA
```

Comment:
Also here, perhaps it would be more self-explanatory to keep `MAX_EXPORT_AREA_SQ_DEG` even if the resulting name `EXPOSURE_MAX_EXPORT_AREA_SQ_DEG` becomes long